crates/runtime/src/memory.rs

```diff
@@ -271,6 +272,12 @@ impl Memory {
     }
 
     fn limit_new(plan: &MemoryPlan, limiter: Option<&mut dyn ResourceLimiter>) -> Result<()> {
+        if plan.memory.minimum > WASM_PAGE_SIZE - 1 {
```

Comment:
Could this be tagged with a `FIXME` as well? I also think it's fine to change this to `==` since that's the only case we're specifically disallowing here, any other cases should be disallowed normally somewhere else.